src/optimization/utils/constraints.jl

```diff
@@ -152,8 +163,10 @@ end
 function encode_relu(::TriangularRelaxedLP, model, ẑᵢⱼ, zᵢⱼ, l̂ᵢⱼ, ûᵢⱼ)
     if l̂ᵢⱼ > 0.0
         @constraint(model, zᵢⱼ == ẑᵢⱼ)
+        @constraint(model, δᵢⱼ == 1)
```

Comment:
must delete (and the ==0 case below as well)

Reply:
ya deleted